src/scores.js

```diff
@@ -1,40 +1,50 @@
+import { postScore, getData } from './apimanagement.js';
+
 const scoreList = document.querySelector('.score-list');
 const scoreForm = document.querySelector('.score-form');
 
 export class Leaderboard {
-    data = []
-
-    addData(s) {
-      this.data.push(s);
-    }
-
-    createScore(nameParam, scoreParam, ind = 0) {
-      const scoreObj = {
-        index: ind,
-        name: nameParam,
-        score: scoreParam,
-      };
-      this.addData(scoreObj);
-    }
+  /* eslint-disable class-methods-use-this */
```

Comment:
Great work @ulises2607 🤝

- [ ] Kindly remove this linter disabling line of code as linter checks help us write more professional code.